README.md

```diff
@@ -2,8 +2,8 @@
 
 <!-- TODO: Create an issue for both, if necessary.
 [![Go Report Card](https://goreportcard.com/badge/github.com/HewlettPackard/lustre_exporter)](https://goreportcard.com/report/github.com/HewlettPackard/lustre_exporter)
-[![Build Status](https://travis-ci.org/HewlettPackard/lustre_exporter.svg?branch=master)](https://travis-ci.org/HewlettPackard/lustre_exporter)
 -->
+[![Continuous Integration](https://github.com/benibr/lustre_exporter/actions/workflows/ci.yml/badge.svg?branch=master)](https://github.com/benibr/lustre_exporter/actions/workflows/ci.yml)
```

Comment:
Can you please use an URL that points to the main repo?

```suggestion
[![Continuous Integration](https://github.com/GSI-HPC/lustre_exporter/actions/workflows/ci.yml/badge.svg?branch=master)](https://github.com/GSI-HPC/lustre_exporter/actions/workflows/ci.yml)
```

Reply:
of course, stupid me :-) done